tools/wltests/test_series

```diff
@@ -385,8 +377,28 @@ energy_measurement:
     # set them here:
     iio_devices: [$IIO_DEVICES]
 EOF
+}
+
+# Prepare WA_AGENDA (if specified it override the TEST_CMD)
+if [ ! -z $WA_AGENDA ]; then
+	if [ ! -f $WA_AGENDA ]; then
+		echo
+		c_error "The \$WA_AGENDA|--wa_agenda is not a valid path for a WA agenda"
+		list_available_agendas
+		exit $INVAL
+	fi
 	TEST_CMD_RESULTS="$RESULTS/wa.\${COMMIT_SHA1:0:7}_\${COMMIT_NAME}"
-	TEST_CMD="wa run -f -d \"$TEST_CMD_RESULTS\" -c \"$WA_LOCAL_CONF\" \"$WA_AGENDA\""
+
+	# Configure WA command
+	# Depending on the EnergyMeter in use, an auxiliary configuration file
+	# can be required.
+	if [ "x$EMETER" == "xACME" ]; then
+		config_emeter_acme
+		TEST_CMD="wa run -f -d \"$TEST_CMD_RESULTS\" -c \"$WA_LOCAL_CONF\" \"$WA_AGENDA\""
```

Comment:
This contains the name of the ADB device in addition to the ACME conf. I guess we should specify it for Moonsoon as well